nrf_802154_sl/CMakeLists.txt

```diff
@@ -0,0 +1,60 @@
+#
+# Copyright (c) 2020 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+# Calculates relative path to the library to be uses taking into account
+# build type of the library, SoC type, floating point ABI, supported FEM type.
+# Note: Subdirectory layout is specific to nrf_802154_sl lib
+function(calculate_lib_path lib_path)
+  # Select build type of the library to link with
+  set(build_type_dir "Release")
+
+  # Select SoC
+  if(CONFIG_SOC_NRF52840)
```

Comment:
What is the reason why we need separate libraries for separate SoCs? Other libraries in this repo fit all SoCs

Reply:
We have some timing-related constants in these libraries (computation duration). The timings differ between SoCs. In the future we plan to replace timing-related constants with TIMER and PPIs. Then we'll deliver like other libraries.